src/kagglehub/auth.py

```diff
@@ -1,2 +1,13 @@
+import json
+from kagglehub.config import set_kaggle_credentials
+
+
 def login():
-    raise NotImplementedError()
+    """Prompt the user for their Kaggle username and API key and save them globally."""
+
+    username = input("Enter your Kaggle username: ")
+    api_key = input("Enter your Kaggle API key: ")
+
+    set_kaggle_credentials(username=username, api_key=api_key)
+
+    print("You are now logged in to Kaggle Hub.")
```

Comment:
We use `logging` instead of `print` statement to allow users to turn them off if they want to.

Example: https://github.com/Kaggle/kagglehub/blob/6e830e666fa8288b1853707818a3cbdd7a308f5b/src/kagglehub/clients.py#L58